### ISSUE TITLE
Error during Invidious installation

### ISSUE THREAD

**Issue by codingnooob** [tmiland/Invidious-Updater#88]
**Describe the bug**
Build process for Invidious install fails because of error, probably due to 
`fatal: detected dubious ownership in repository at '/home/invidious/invidious'` (see screenshot)

I tried doing `git config --global --add safe.directory /home/invidious/invidious`, removing, and reinstalling Invidious with option 7, but ended up with the exact same result 

**Expected behavior**
Invidious gets installed

**Screenshots**
![2022-07-23_22-52](https://user-images.githubusercontent.com/66186626/180634364-bc271c83-debe-4526-b99c-a482cf1446f3.png)

**Desktop**
 - OS: Pop!_OS 22.04 LTS 64-bit

**Comment by tmiland** [comment]
That should have been fixed in latest [Master](https://github.com/tmiland/Invidious-Updater/commit/9137a0681e2ecf7680d582302a49aba381370ba5)

```
$ wget https://github.com/tmiland/Invidious-Updater/raw/master/invidious_update.sh
$ chmod +x invidious_update.sh
$ ./invidious_update.sh
```

Please try and report back.

Thanks.

**Comment by mokahless** [comment]
Different user reporting in that this is not fixed in 1.6.3 from apt. Same issue with the exact same message on ubuntu 22 (latest updates as of today's date). Running as an unprivileged container under proxmox (if that's somehow related, sorry for the bump but I don't see any clues as to that being the issue).

I've installed this via apt, so checked /opt/invidious-updater-linux_amd64/invidious_update.sh and it has x permissions.

So I attempted to download the script and run it manually as you described and it refuses to install, telling me it is already installed.

Strangely, when I gave it option 10 after giving up, instead of exiting, it said "let's go through some configuration options" then told me it was "ready to be installed" and pressing enter compiles it. Since compiling is time consuming, this definitely didn't happen before with the initial error. Back to the main menu, entering 10 after this does exit and it appears successfully installed.
